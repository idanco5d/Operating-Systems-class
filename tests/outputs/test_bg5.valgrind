==28037== Memcheck, a memory error detector
==28037== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==28037== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==28037== Command: /mnt/c/Users/illus/os/hw/1/smash
==28037== Parent PID: 28031
==28037== 
==28037== 
==28037== FILE DESCRIPTORS: 4 open (3 std) at exit.
==28037== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_bg5.valgrind
==28037==    <inherited from parent>
==28037== 
==28037== 
==28037== HEAP SUMMARY:
==28037==     in use at exit: 0 bytes in 0 blocks
==28037==   total heap usage: 105 allocs, 105 frees, 83,546 bytes allocated
==28037== 
==28037== All heap blocks were freed -- no leaks are possible
==28037== 
==28037== For lists of detected and suppressed errors, rerun with: -s
==28037== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
