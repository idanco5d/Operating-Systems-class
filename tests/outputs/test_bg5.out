smash> smash: got ctrl-Z
smash: process 8906 was stopped
smash> smash: got ctrl-Z
smash: process 8951 was stopped
smash> smash: got ctrl-Z
smash: process 8994 was stopped
smash> smash: got ctrl-Z
smash: process 9070 was stopped
smash> smash: got ctrl-Z
smash: process 9087 was stopped
smash> [1] sleep 100 : 8906 2 secs (stopped)
[2] sleep 200 : 8951 1 secs (stopped)
[3] sleep 300 : 8994 1 secs (stopped)
[4] sleep 400 : 9070 0 secs (stopped)
[5] sleep 500 : 9087 0 secs (stopped)
smash> sleep 300 : 1
smash> sleep 400 : 0
smash> sleep 200 : 1
smash> sleep 500 : 0
smash> sleep 100 : 2
smash> [1] sleep 100 : 8906 2 secs
[2] sleep 200 : 8951 1 secs
[3] sleep 300 : 8994 1 secs
[4] sleep 400 : 9070 0 secs
[5] sleep 500 : 9087 0 secs
smash> smash: sending SIGKILL signal to 5 jobs:
8906: sleep 100
8951: sleep 200
8994: sleep 300
9070: sleep 400
9087: sleep 500
